apps/auth_zero/tests.py

```diff
@@ -0,0 +1,31 @@
+from django.test import TestCase
+from django.contrib.auth import get_user_model
+from common.config import ConfigFileLoader, CredentialsLoader
+
+
+class UserTestCase(TestCase):
+    def setUp(self):
+        self.user_model = get_user_model()
+        config_file = "apps/auth_zero/config/test_config.ini"
```

Comment:
Create data/config/test/test_config.ini instead of apps/auth_zero/config/test_config.ini

Reply:
what i mean by that is apps/auth_zero/config/data/test/test_config.ini instead of